src/linux/install-tools.sh

```diff
@@ -54,7 +54,10 @@ install_tools() {
   cp -r ./usr/sbin/mdata-* /usr/sbin/
   cp -r ./usr/share/man/man1/mdata-* /usr/share/man/man1/
   ln -fs /usr/sbin/mdata-get /lib/smartdc/mdata-get
-  mv /etc/rc.local /etc/rc.local-backup
+  if [[ -e /etc/rc.local ]]
```

Comment:
This is a nice addition! However I'd prefer this to be consistent to how we typically format test blocks, e.g.,:

```
if [[ -e /etc/rc.local ]] ; then
```